src/ncurses.cr

```diff
@@ -166,5 +169,17 @@ module NCurses
     @@initialized = false
   end
 
+  def ncurses_bits(mask, shift)
```

Comment:
I don't think those methods should be exposed directly under the `NCurses` namespace.